workload/src/main/java/com/google/finapp/WorkloadClient.java

```diff
@@ -0,0 +1,59 @@
+package com.google.finapp;
+
+import com.google.finapp.FinAppGrpc.FinAppBlockingStub;
+import com.google.protobuf.ByteString;
+import io.grpc.ManagedChannel;
+import io.grpc.StatusRuntimeException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class WorkloadClient {
+
+  private final ManagedChannel channel;
+  private static Logger logger = Logger.getLogger(WorkloadClient.class.getName());
+
+  private WorkloadClient(ManagedChannel channel) {
+    this.channel = channel;
+  }
+
+  public static WorkloadClient getWorkloadClient(ManagedChannel channel) {
+    return new WorkloadClient(channel);
+  }
+
+  public ByteString createAccount(
+      String balance, CreateAccountRequest.Type type, CreateAccountRequest.Status status) {
+    FinAppBlockingStub blockingStub = FinAppGrpc.newBlockingStub(channel);
+    CreateAccountRequest request =
+        CreateAccountRequest.newBuilder()
+            .setBalance(balance)
+            .setType(type)
+            .setStatus(status)
+            .build();
+    try {
+      CreateAccountResponse response = blockingStub.createAccount(request);
+      logger.log(Level.INFO, String.format("Account created %s", response));
+      return response.getAccountId();
+    } catch (StatusRuntimeException e) {
+      logger.log(Level.SEVERE, String.format("Error creating account %s", request));
+      return null;
```

Comment:
I have the code not error out when there's an exception creating an account. This is so that if there's some small error or something and only one or a small amount of `createAccount`s fails, the workload can continue regardless. 

This can be pretty easily changed to make the code fail automatically. Let me know your thoughts.

Reply:
This `null` is intercepted in the `WorkloadGenerator` and essentially ignored.